.pre-commit-config.yaml

```diff
@@ -55,7 +55,7 @@ repos:
             pyarrow>=1.0.1,
             chardet>=3.0.4,
             fastavro>=1.0.0.post1,
-            python-snappy>=0.5.4,
+            cramjam>=2.7.0,
```

Comment:
I think this should be reverted 🤔

Reply:
should match `requirements.txt`

Reply:
python-snappy>=0.7.1 supports Python 3.11 by switching over to cramjam and removing libsnappy-dev dependency (I contributed https://github.com/intake/python-snappy/pull/130)

I will work dependencies in a minute

Reply:
we reverted these cramjam changes due to failing tests locally though in #1122. 
- do we have to include cramjam + 3.11 in same pr? 
- I'm game for the 3.11 add
- is the test suite running locally for you using `DATAPROFILER_SEED=0 python3 -m unittest discover -p "test*.py"` and `tox -e py311`?